public/content/documentation/web/component/strikethrough-content.md

```diff
@@ -6,75 +6,94 @@ How to test a strikethrough element
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a strikethrough element
 
 1. Test keyboard only, then screen reader + keyboard actions
 
-   - Skip-links: Focus moves directly to the header or navigation
-
-   - Tab: Nothing, headings are not focusable unless they are actionable
-
-   - Arrow-keys: Headings are browsed
+      - Arrow: Browses the content
 
 2. Test mobile screenreader gestures
 
-   - Swipe: Focus moves directly to the header or navigation
-
-   - Doubletap: This typically activates most elements
+      - Swipe: The content makes sense and is in logical order
 
 3. Listen to screenreader output on all devices
 
-   - It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - Group: It typically contains the name and primary navigation of the website
+      - Name: The content makes sense and is in logical order
 
 Full information: [https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/strikethrough-content](https://www.magentaa11y.com/MagentaA11yV2#/web-criteria/component/strikethrough-content)
 
 ## Gherkin
 
 ### #a11y - Web Accessibility Acceptance Criteria
 
-How to test a header
+How to test a strikethrough element
 
 GIVEN THAT I am on a page with a header landmark
 
 1. Keyboard for mobile & desktop
 
-   - WHEN I use the tab key to enter the web browser window I SEE focus is strongly visually indicated on interactive components
+      - WHEN I use the arrow key to browse the content I SEE the content comes into view
 
 2. Desktop screenreader
 
-   - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND
-
-   - I use the tab key to enter the web browser window
-
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - I HEAR It typically contains the name and primary navigation of the website
+      - WHEN I use a desktop screenreader (NVDA, JAWS, VoiceOver) AND 
+      - I use the arrow key to browse the content
+         - I HEAR the content makes sense and is in logical order
 
 3. Mobile screenreader
 
-   - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
-
-   - I swipe to focusable elements in the header
-
-   - I HEAR It is discoverable with screenreader shortcuts as header/banner landmark
-
-   - I HEAR It typically contains the name and primary navigation of the website
-
+      - WHEN I use a mobile screenreader (Talkback, VoiceOver) AND
+      - I swipe to browse the content
+         - I HEAR the content makes sense and is in logical order
```

Comment:
Plus one for "in logical order" here; could we put "a" ahead of "logical order" so it reads a bit clearer?